packages/babel-plugin-emotion/src/utils/get-styled-options.js

```diff
@@ -2,22 +2,35 @@
 import { getLabelFromPath } from './label'
 import { getTargetClassName } from './get-target-class-name'
 
+const getKnownProperties = (path, keyName) =>
```

Comment:
This should be:
```js
const getKnownProperties = path => path.properties.map(p => p.key.name)
```
Right now you always map to `false`, because you are not passing the second argument (which is also not needed here).

I wasnt going to comment on that, because using an array for this is OK, but since you need to change it anyway I would propose changing the return type from an array to a set (and adjusting the consumers of this of course).

`path` is guaranteed to be here, so I don't think there is much need to guard against it being missing right now.

Reply:
🆗